css/design_system/components/card.scss

```
@@ -0,0 +1,114 @@
+ /**
+ * @title Basic Card
+ * @category components
+ * @component_type card
+ * @value s-card
+ * @description The basic card allows for a title, subtitle, content, and footer - all of which are optional. The example below shows a card that does use all options.
+ *
+ * @example
+ * <div class="s-card">
+ *  <div class="s-card-basic-header">
+ *    <h2 class="s-card-title">Card Title</h2>
+ *    <h3 class="s-card-subtitle">Card Subtitle</h3>
+ *  </div> 
+ *  <div class="s-card-content">
+ *    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum risus odio quis sit mattis in luctus. Dignissim auctor aliquam suspendisse non in ac, purus. Aliquam arcu justo quam lorem at id proin morbi. A arcu accumsan, eu id fringilla non.</p>
+ *    <p class="s-card-footer">Footer content</p>
+ *  </div>
+ * </div>
+ */
+
+.s-card {
+  @extend .s-p-4;
+  display: inline-block;
+  border-radius: var(--s-space-1);
+  border-top: 8px solid var(--s-color-cyan-600);
+  box-shadow: 0 1px 3px var(--s-color-gray-400);
+}
+
+.s-card-basic-header {
+  .s-card-title {
+    @extend .s-text-gray-800;
+    @extend .s-text-uppercase;
+    @extend .s-m-0;
+    @extend .s-font-heavy;
+    font-size: var(--s-font-size-lg); 
+  }
+
+  .s-card-subtitle {
+    @extend .s-text-gray-500;
+    margin-top: var(--s-space-1);
+    letter-spacing: 0.75px;
+    font-size: 1.25rem;
```

Let's bump this down a notch so we can use a standard size.
```suggestion
    font-size: var(--s-font-size-md);
```